libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/Kapt3KotlinGradleSubplugin.kt

```diff
@@ -74,12 +74,20 @@ class Kapt3GradleSubplugin @Inject internal constructor(private val registry: To
         fun getKaptGeneratedKotlinSourcesDir(project: Project, sourceSetName: String) =
             File(project.buildDir, "generated/source/kaptKotlin/$sourceSetName")
 
-        private val VERBOSE_OPTION_NAME = "kapt.verbose"
-        private val USE_WORKER_API = "kapt.use.worker.api"
-        private val INFO_AS_WARNINGS = "kapt.info.as.warnings"
-        private val INCLUDE_COMPILE_CLASSPATH = "kapt.include.compile.classpath"
-        private val INCREMENTAL_APT = "kapt.incremental.apt"
-        private val KAPT_KEEP_KDOC_COMMENTS_IN_STUBS = "kapt.keep.kdoc.comments.in.stubs"
+        // Pairs of names and default values of properties
+        // The default values should typically match those defined in org.jetbrains.kotlin.base.kapt3.KaptFlag.
+        private val VERBOSE_OPTION_NAME = Pair("kapt.verbose", false)
```

Comment:
VERBOSE_OPTION_NAME -> VERBOSE_OPTION?

Reply:
Yup, I've changed it to `KAPT_VERBOSE` and prefixed all options with `KAPT_` for them to be consistent .